src/uikit/Footer.tsx

```diff
@@ -2,56 +2,104 @@ import {
   Box,
   Container,
   Image,
+  Link,
   SimpleGrid,
   Stack,
   Text,
 } from '@chakra-ui/react';
 
+import { publicRuntimeConfig } from '../../next.config';
+
+const plugins = [
+  {
+    title: 'Documentation',
+    href: publicRuntimeConfig.docsUrl,
+  },
+  {
+    title: 'Support Us',
+    href: publicRuntimeConfig.supportUrl,
+  },
+];
+
+const socials = [
+  {
+    title: 'Discord',
+    href: publicRuntimeConfig.discordServerUrl,
+  },
+  {
+    title: 'Github',
+    href: publicRuntimeConfig.githubUrl,
+  },
+];
+
+type SubTitleProps = {
+  title: string;
+};
+
+function SubTitle({ title }: SubTitleProps) {
+  return (
+    <Text
+      color='white'
+      fontWeight='medium'
+      fontSize='sm'
+      letterSpacing='10px'
+      textTransform='uppercase'>
+      {title}
+    </Text>
+  );
+}
+
 export default function Footer() {
   const currentYear = new Date().getFullYear();
+
   return (
-    <Box color='white'>
-      <Container as={Stack} maxW='6xl' py={10}>
+    <Box background='black'>
+      <Container as={Stack} maxW='6xl' py='16' px='10'>
```

Comment:
Same comment about sizing rules before, we have inconsistency about value. I think we should use the same rules, what do you think?

Reply:
updated